hack/test-cmd.sh

```diff
@@ -238,6 +238,9 @@ fi
 PATH="${KUBE_OUTPUT_HOSTBIN}":$PATH
 
 kube::log::status "Checking kubectl version"
+
+# Note that this fails on a dirty commit.  To avoid that from happening, .gitignore
```

Comment:
not sure: only way to find out is to run experiments against the CI I think ?  I can't promise that dirty commits are still an issue, since these build scripts changed a few times and also since the ci tooling may have changed .  So I think best to take a mental note and proceed with the possibility that various build tasks may be called in different orders....